packages/ui/src/lib/table/Table.spec.ts

```diff
@@ -265,3 +270,29 @@ test('Expect update callback', async () => {
 
   expect(callback).toHaveBeenCalled();
 });
+
+test('Expect table to be sorted by Id on load', async () => {
+  render(TestTable, {});
+
+  // Wait for the table to load and fetch all rows
+  const rows = await screen.findAllByRole('row');
+  expect(rows).toBeDefined();
+  expect(rows.length).toBe(4);
+
+  const headers = await screen.findAllByRole('columnheader');
+  expect(headers).toBeDefined();
+  expect(headers.length).toBe(6);
+  expect(headers[2].textContent).toContain('Id');
+
+  // onMount will wait for tick() to complete, so we need to wait for the sort
+  while (!headers[2].innerHTML.includes('fa-sort-up')) {
```

Comment:
I remember there is a better way to wait for an event but can't find the name but @dgolovin may help